javascript/compute-height.js

```diff
@@ -114,8 +112,8 @@ function getComfyuiContainer() {
     return document.getElementById("comfyui_webui_container") ?? null;
 }
 
-function getComfyuiIFrameElement() {
-    return document.querySelector('comfyui_general_tab') ?? null;
+function getWorkflowIdsElement() {
+    return document.getElementById('comfyui_workflow_id_list') ?? null;

```

Comment:
same again, I would call this `getWorkflowTypeIdsElement` or leave it as `getIFrameIdsElement`